src/platform/loader-web.ts

```diff
@@ -69,7 +69,7 @@ export class Loader extends LoaderBase {
       // import (execute) particle code
       importScripts(url);
     } catch (e) {
-      e.message = `Error loading Particle from '${path}': ${e.message}`;
+      warn(`Error loading Particle from '${path}': ${e.message}`);
```

Comment:
changed this, because it complained about `e` being readonly ...

Reply:
Oops, I should have caught that. Can you change it to:
`throw new Error(`Error loading Particle from '${path}': ${e.message}`);`

Reply:
... unless you actually think it's better to `warn` there, whatever you think is best.

Reply:
updated to `throw new...` (don't know why i changed it in the first place)